ichnaea/data/area.py

```diff
@@ -43,11 +43,13 @@ def region(self, ctr_lat, ctr_lon, mcc, cells):
             for reg in regions:
                 grouped_regions[reg] += 1
             max_count = max(grouped_regions.values())
-            max_regions = sorted(
-                [k for k, v in grouped_regions.items() if v == max_count]
-            )
+            max_regions = [k for k, v in grouped_regions.items() if v == max_count]
             # If we get a tie here, randomly choose the first.
-            region = max_regions[0]
+            if None in max_regions:
+                # Emulate Python 2.6, where None is sorted first
+                region = None
+            else:
+                region = sorted(max_regions)[0]
```

Comment:
Why not use a key? Something like this:

```
region = sorted(max_regions, key=lambda item: "" if region is None else region)
```

Reply:
Or more korrect:

```
region = sorted(max_regions, key=lambda item: item or "")
```

Reply:
I feel weird using ``key`` to work around this change, but it does make it a one-line change, and could be use other places in the code where ``sorted``/``sort`` breaks unexpectedly.